src/brouwers/shop/templates/shop/includes/sidenav.html

```diff
@@ -0,0 +1,21 @@
+{% load i18n %}
+<ul class="category-sidenav">
+    <li class="category-sidenav__header">
+        {% trans "Categories" %}
+    </li>
+    {% for cat in categories %}
+        <li class="category-sidenav__item">
+            <a class="category-sidenav__link {% if cat.id == category.id %}category-sidenav__link--active{% endif %}"
+               href="{{ cat.get_absolute_url }}">{{ cat.name }}</a>
+            {% if cat.id == category.id and not cat.is_leaf or cat.id == category.get_root.id %}
```

Comment:
the `category.get_root` call is being repeated in the for-loop and generating a query for every `cat` in `categories`. Can you move this out of the forloop?

You can do something like:

```
{% with root_id=category.get_root.id %}
    {% for cat in categories %}
    ...
    {% endfor %}
{% endwith %}
```